packages/jest-gas-reporter/.eslintrc.js

```diff
@@ -1,178 +1,16 @@
-  radix: 'error',
-  'space-before-function-paren': 'off',
-  'spaced-comment': 'error',
-  'use-isnan': 'error',
-  'valid-typeof': 'off',
-  'prettier/prettier': 'error',
-  'no-unused-vars': 'off',
-  'import/default': 'off',
-};
-
-const TSRules = {
-  '@typescript-eslint/adjacent-overload-signatures': 'error',
-  '@typescript-eslint/array-type': 'error',
-  '@typescript-eslint/ban-types': 'error',
-  '@typescript-eslint/class-name-casing': 'error',
-  '@typescript-eslint/consistent-type-assertions': 'error',
-  '@typescript-eslint/consistent-type-definitions': 'off',
-  '@typescript-eslint/explicit-member-accessibility': [
-    'off',
-    {
-      accessibility: 'explicit',
-    },
-  ],
-  '@typescript-eslint/indent': 'off',
-  '@typescript-eslint/interface-name-prefix': 'off',
-  '@typescript-eslint/member-delimiter-style': [
-    'off',
-    'error',
-    {
-      multiline: {
-        delimiter: 'none',
-        requireLast: true,
-      },
-      singleline: {
-        delimiter: 'semi',
-        requireLast: false,
-      },
-    },
-  ],
-  '@typescript-eslint/member-ordering': 'error',
-  '@typescript-eslint/no-empty-function': 'error',
-  '@typescript-eslint/no-empty-interface': 'error',
-  '@typescript-eslint/no-explicit-any': 'off',
-  '@typescript-eslint/no-misused-new': 'error',
-  '@typescript-eslint/no-namespace': 'error',
-  '@typescript-eslint/no-parameter-properties': 'off',
-  '@typescript-eslint/no-use-before-declare': 'off',
-  '@typescript-eslint/no-var-requires': 'error',
-  '@typescript-eslint/prefer-for-of': 'off',
-  '@typescript-eslint/prefer-function-type': 'error',
-  '@typescript-eslint/prefer-namespace-keyword': 'error',
-  '@typescript-eslint/quotes': 'off',
-  '@typescript-eslint/semi': ['off', null],
-  '@typescript-eslint/space-within-parens': ['off', 'never'],
-  '@typescript-eslint/triple-slash-reference': 'error',
-  '@typescript-eslint/type-annotation-spacing': 'off',
-  '@typescript-eslint/unified-signatures': 'error',
-  '@typescript-eslint/explicit-function-return-type': 'off',
-  '@typescript-eslint/no-unused-vars': 'off',
-  '@typescript-eslint/ban-ts-ignore': 'off',
-  '@typescript-eslint/no-use-before-define': 'off',
-};
-
 module.exports = {
   env: {
     node: true,
-    es6: true,
+    es6: true
   },
   parser: '@typescript-eslint/parser',
-  parserOptions: {
-    project: 'tsconfig.json',
-    ecmaVersion: 2018,
-    sourceType: 'module',
-  },
-  plugins: ['@typescript-eslint', '@typescript-eslint/tslint', 'prettier'],
+  plugins: ['@typescript-eslint', 'prettier'],
   extends: [
-    'plugin:@typescript-eslint/recommended',
     'eslint:recommended',
-    'plugin:import/errors',
-    'plugin:import/warnings',
-    'plugin:import/typescript',
-    'prettier/@typescript-eslint',
-    'prettier',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:prettier/recommended'
   ],
-  rules: Object.assign(rules, TSRules),
+  rules: {
+    'prettier/prettier': ['warn']
```

Comment:
Hmm why not error?

Reply:
To distinguish between linting issues and build issues. I think it makes the code more readable.

Reply:
The question is do we want CI to fail on formatting issues?
Anyways your choice :)

Reply:
Good point. Turns out we can have the best of both worlds with `--max-warnings=0`.

<img width="315" alt="Screen Shot 2019-12-06 at 10 50 08" src="https://user-images.githubusercontent.com/1933029/70335970-5b664e80-1816-11ea-8aa0-d0b7889c6b2a.png">
<img width="647" alt="Screen Shot 2019-12-06 at 10 50 25" src="https://user-images.githubusercontent.com/1933029/70335971-5b664e80-1816-11ea-88dc-d774c5629aab.png">